Automattic/wp-calypso
# Add 1TB backup storage feature to Jetpack Complete product card

#### Changes proposed in this Pull Request
Recently, we've introduced 1TB backup storage in the Jetpack Complete plan. We want to inform users about the change. Thus, we add a new feature point to product cards.

#### Testing instructions

1. Start a local development environment with: `yarn start` (calypso) and `yarn start-jetpack-cloud-p` (jetpack cloud)
2. Visit page: http://jetpack.cloud.localhost:3001/pricing
3. Verify that Jetpack Complete product card has the mentioned feature "1TB of backup storage"
<img width="1489" alt="Screenshot 2022-05-19 at 13 46 28" src="https://user-images.githubusercontent.com/8419292/169289731-34d75d43-2167-4a80-be74-8037b03d0c1c.png">

4. Visit page: http://calypso.localhost:3000/jetpack/connect/store
5. Verify that Jetpack Complete product card has the mentioned feature "1TB of backup storage"
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/8419292/169290053-59b56d29-8539-43a8-96c9-7f801046c351.png">
